CHANGELOG.md

```diff
@@ -31,6 +31,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 - Fix the empty output of `go.opentelemetry.io/otel/log.Value` in `go.opentelemetry.io/otel/exporters/stdout/stdoutlog`. (#5311)
 
+### Fixed
```

Comment:
Looks like this duplicates the other "fixed" section

Reply:
Ah yes, GitHub's auto-merging was wrong there.